backend/internal/domain/repository/post_repository.go

```diff
@@ -12,4 +12,5 @@ type PostRepository interface {
 	GetByID(ctx context.Context, id int) (*model.Post, error)
 	Create(ctx context.Context, post *model.Post) (*model.Post, error)
 	Update(ctx context.Context, post *model.Post) (*model.Post, error)
+	Delete(ctx context.Context, postId int) error
```

Comment:
無茶苦茶細かいけど、`GetById`と合わせた方がいいかも！

```suggestion
	Delete(ctx context.Context, id int) error
```

Reply:
[e04d642](https://github.com/givery-bootcamp/dena-2024-team3/pull/82/commits/e04d6424209616aa1b95386222c6c0faf2cb9cb0)で修正しました。